composer.json

```diff
@@ -3,22 +3,29 @@
     "description": "Saxulum Doctrine MongoDB Provider",
     "keywords": ["saxulum", "pimple", "cilex", "silex", "doctrine", "mongo", "mongodb"],
     "license": "MIT",
+    "type": "library",
     "authors": [
         {
             "name": "Dominik Zogg",
             "email": "dominik.zogg@gmail.com"
         }
     ],
+    "replace": {
+        "ext-mongo": "1.6.12"
+    },
     "require": {
-        "php": ">=5.3.3",
-        "doctrine/mongodb": "~1.1",
-        "pimple/pimple": ">=2.1,<4",
-        "psr/log": "1.0.*"
+        "php": ">=5.5",
```

Comment:
yes, "~5.5|~7.0" should a better match for our version requirement - and yes the provider changes will work on php >=5.4.n/5.5.n/5.6.n (but won't work with <= 5.3)